test-data/unit/fine-grained-modules.test

```diff
@@ -1271,12 +1271,12 @@ a.py:2: error: Too many arguments for "foo"
 
 [case testAddModuleAfterCache3-only_when_cache]
 # cmd: mypy main a.py
-# cmd2: mypy main a.py b.py c.py d.py e.py f.py g.py
-# cmd3: mypy main a.py b.py c.py d.py e.py f.py g.py
+# cmd2: mypy main a.py b.py c.py d.py e.py f.py g.py h.py
+# cmd3: mypy main a.py b.py c.py d.py e.py f.py g.py h.py
```

Comment:
What's this change?

Reply:
It tests that we are abandoning cache if more than half of files are missing.